apps/docs/src/app/core/component-docs/file-uploader/file-uploader-docs.component.html

```diff
@@ -0,0 +1,95 @@
+<fd-docs-section-title [id]="'file-uploader-simple'" [componentName]="'file-uploader'">
+    Simple File Upload
+</fd-docs-section-title>
+<description>Used in this context, it will only allow a set of extensions to be added.</description>
+<component-example>
+    <fd-file-uploader-example></fd-file-uploader-example>
+</component-example>
+<code-example [exampleFiles]="fileUpladerExample"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title [id]="'file-uploader-truncation'" [componentName]="'file-uploader'">
+    File Uploader Truncation example
+</fd-docs-section-title>
+<description>Long placeholder for the input element will be truncated.</description>
+<component-example>
+    <fd-file-uploader-truncation-example></fd-file-uploader-truncation-example>
+</component-example>
+<code-example [exampleFiles]="fileUpladerExample"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title [id]="'file-uploader-compact'" [componentName]="'file-uploader'">
+    Compact File Upload
+</fd-docs-section-title>
+<description>Example to demonstrate the compact file uploader. </description>
```

Comment:
Say what api must be used to achieve compact

Reply:
I have mentioned it in the description now.